config/initializers/omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :github, ENV["GITHUB_CLIENT_ID"], ENV["GITHUB_CLIENT_SECRET"], scope: "user:email"
+end
```

Comment:
it's showing me a warning in github that there's no extra newline at the end - I don't know if that's a problem or just ruby style but we might have to add a newline at the end - you can set up VS code to automatically add newlines at the ends of files, I can't remember how but we can look into it if you need